tests/e2e/users.feature

```diff
@@ -151,6 +152,7 @@ Feature: Users & Permissions
     And I should see "Suggested secure password"
 
   @javascript
+      @foo
```

Comment:
@bobdenotter Oops... think we have to remove that! 🤪

Reply:
Removed it! 🙈